x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_import_export/basic_license_essentials_tier/import_rules.ts

```diff
@@ -17,7 +17,7 @@ export default ({ getService }: FtrProviderContext): void => {
   const securitySolutionApi = getService('securitySolutionApi');
   const log = getService('log');
 
-  describe('@ess @serverless import_rules', () => {
+  describe('@ess @serverless @serverlessQA import_rules', () => {
```

Comment:
What's the reason to add `import_rules` to `ServerlessKibanaQAGate`, but not `import_rules_with_overwrite`?

I'm just thinking that the import function is becoming increasingly important, especially since it will be the main way to bulk create rules once the Bulk Create API is fully deprecated in 9.0

Reply:
The goal is to enable smoke testing for importing functionality. Do you a scenario in mind when `import_rules_with_overwrite` fails only in MKI while passes in PR and on merge pipelines but `import_rules` passes everywhere?